packages/apps/docs/src/components/BrowseSection/BrowseSection.tsx

```diff
@@ -64,7 +64,7 @@ const BrowseSection: BrowseSectionType = ({
             !React.isValidElement(child) ||
             (child.type !== LinkBlock &&
               child.type !== NextLink &&
-              child.type !== Link &&
+              child.type !== Button &&
```

Comment:
just for my understanding because a link and a button have 2 different functionalities is this button triggering an action or just a link?

Reply:
This is the only use case I was having doubts on this implementation. The button behaves as a link if you pass an href, if an onPress is provided it's an action on the page. 

I am missing context about this component to be sure if this is correct behaviour. I'm looking for feedback on this.